packages/ts/react-grid/src/autogrid.tsx

```diff
@@ -126,6 +127,11 @@ function useColumns(
       </ColumnContext.Provider>
     );
   });
+  if (options.customColumns) {
+    return [...autoColumns, ...options.customColumns];
```

Comment:
Would the way of customizing order of columns work for the custom columns a well?

Reply:
I am not sure how it should work as the custom column does not have an id, at least right now

Reply:
Is the custom columns able to add the functionality of combining two values such as `first name` + `last name` and showing it under the `full name` column?

Reply:
For that case, something more would be needed. For the delete case, it is fine to only be able to add columns last